common/sai.py

```diff
@@ -175,8 +175,16 @@ def switch_oid(self, value):
         self.create_alias('SWITCH_ID', 'SAI_OBJECT_TYPE_SWITCH', value)
         self._switch_oid = value
 
-    def process_commands(self, commands):
-        yield from map(self.command_processor.process_command, commands)
+    def process_commands(self, commands, cleanup=False):
+        _commands = []
+        if cleanup:
+            for command in reversed(commands):
+                if command['op'] in ['create', 'remove']:
```

Comment:
Why not only select `create` commands? I assume if you processed a remove command earlier, it makes no sense to repeat it in cleanup.